validations/applications.js

```diff
@@ -0,0 +1,16 @@
+const requireValidation = (req, res, next) => {
+  if (!req.body.positions) {
+    return res.status(400).send('Id position is required');
+  }
+  if (!req.body.postulants) {
+    return res.status(400).send('Id postulant is required');
+  }
+  if (!req.body.result) {
+    return res.status(400).send('Result is required');
+  }
```

Comment:
you should check the format of each body value

Reply:
Where do I have to validate the format of each value? Inside of every require?? Or do I have to create another validation only for the format?? And, in which format I do that?? Sorry it's a lot of questions..

Reply:
In my opinion, it would validate the required values and the formats in different functions. In this way, you can reuse the format validate function.